meilisearch-index-setting-macro/Cargo.toml

```diff
@@ -1,7 +1,9 @@
 [package]
 name = "meilisearch-index-setting-macro"
 version = "0.1.0"
+description = "Helper tool to generate settings of an index"
```

Comment:
Maybe we should precise the "Meilisearch" on it since it will be available for everyone...

```suggestion
description = "Helper tool to generate settings of an Meilisearch index"
```